android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/UpdatesBinding.kt

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020-present 650 Industries. All rights reserved.
+package versioned.host.exp.exponent.modules.universal
+
+import android.content.Context
+import javax.inject.Inject
+import host.exp.exponent.ExpoUpdatesAppLoader
+import expo.modules.updates.UpdatesConfiguration
+import expo.modules.updates.UpdatesInterface
+import expo.modules.updates.UpdatesService
+import expo.modules.updates.db.DatabaseHolder
+import expo.modules.updates.selectionpolicy.SelectionPolicy
+import expo.modules.updates.loader.FileDownloader
+import expo.modules.updates.db.entity.UpdateEntity
+import expo.modules.updates.db.entity.AssetEntity
+import expo.modules.updates.launcher.Launcher.LauncherCallback
+import host.exp.exponent.kernel.KernelProvider
+import host.exp.exponent.di.NativeModuleDepsProvider
+import host.exp.exponent.kernel.KernelConstants
+import java.io.File
+
+class UpdatesBinding(context: Context, experienceProperties: Map<String, Any?>) :
+  UpdatesService(context), UpdatesInterface {
+
+  @Inject
+  lateinit var mDatabaseHolder: DatabaseHolder
```

Comment:
```suggestion
  lateinit var databaseHolder: DatabaseHolder
```

Reply:
Needs to be something other than databaseHolder not to conflict, I chose `databaseHolderInternal`.